src/gui/gui_simulation.cpp

```diff
@@ -51,6 +52,16 @@ void DissolveWindow::setupIteration(int count)
     // Clear the messages tab
     clearMessages();
 
+    // Check if pair potential range is too large for configuration box, ask to auto adjust if so
+    if (PairPotentialCheck::checkPairPotential(dissolve_.coreData().configurations(), dissolve_.pairPotentialRange())) {
+
+        if (PairPotentialCheck::displayPairPotentialWarning()) {
+
+            dissolve_.setPairPotentialRange(PairPotentialCheck::radius);
+            Messenger::print("Pair potential range updated to {}", PairPotentialCheck::radius);
+        }
+    }
```

Comment:
Moving away from the separate class, this would become something like:
```suggestion
    // Check if pair potential range is too large for configuration box, ask to auto adjust if so
    checkPairPotentialRange();
```
Furthermore, I now realise that we can only ever make this check just after a new `Configuration` is generated in the GUI, either by the user pressing "Generate" or though using the AddConfigurationDialog. The `Dissolve::prepare()` function is the one that sets everything up and fails if, for example, the pair potential range is too long, but by this point it is too late for us to intervene!  So, this bit should actually be removed and we should only check after the other two user-triggered events.